app/ui-react/syndesis/src/modules/data/shared/VirtualizationUtils.ts

```diff
@@ -194,3 +204,67 @@ export function isDvConnectionSelected(conn: Connection) {
   }
   return isSelected;
 }
+
+/**
+ * Generate preview SQL for the specified view definition
+ * @param viewDefinition the ViewDefinition
+ */
+export function getPreviewSql(viewDefinition: ViewDefinition): string {
+  if (viewDefinition) {
```

Comment:
Mmm, reading the type this condition can never be true.

Reply:
Right, I'll change this

Reply:
pushed a change for this